capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -14,6 +14,13 @@ class ClubPage extends Component {
         .then(response => response.json()).then(res => this.setState({club: res[0]}));
   }
 
+  handleClick = (e) => {
+    e.preventDefault();
+    fetch(`/api/clubs?id=${this.props.match.params.id}`, {method: "delete"})
+    .then(response => {console.log(response)});
```

Comment:
This should also handle errors (`.catch`). Consider what you want the front-end to do when the delete fails on the back-end: surface a failure message to the user in the UI? automatically retry the request a certain number of times (bonus points for exponential backoff but not necessary at this stage)? Regardless, you'll want to set up some sort of client-side error logging at some point (where the errors are sent to a centralized place you have access to, not just logged in the client console).

Logging the error in the console and putting a TODO (with an associated issue) here to handle errors properly is fine at this point, just make sure you're thinking about this and document the issue.

Reply:
I looked up frontend logging on GCP and it looks like Winston may be something we can use to send errors similar to the way we send the backend Java errors to the console?
https://cloud.google.com/logging/docs/setup/nodejs#using_winston

Reply:
This is a good thing to think about, and we'll discuss as a team how we want to handle this. In the meantime, I created an issue and marked the TODO here (and below). I also now only push the user to the homepage in the case of a successful delete.